pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -61,12 +61,15 @@ const (
 )
 
 var (
-	// The container that we use to run in the EventListener Pods
-	elImage = flag.String("el-image", "override-with-el:latest",
+	// ELImage defines the container that we use to run in the EventListener Pods
+	ELImage = flag.String("el-image", "override-with-el:latest",
 		"The container image for the EventListener Pod.")
 	// ElPort defines the port for the EventListener to listen on
 	ElPort = flag.Int("el-port", 8080,
 		"The container port for the EventListener to listen on.")
+	// ELSetSecurityContext defines if the security context is set.
+	ELSetSecurityContext = flag.Bool("el-security-context", true,
```

Comment:
I forgot to mention this during WG call 
If we keep default value as true then for `openshift` platform we need to update the controller deployment to set the flag to `false`
I believe which is extra step
so considering this shall we make default as `false`
@dibyom @bigkevmcd @gabemontero WDYT?

Reply:
Yeah, my understanding is that openshift already generates/modifies some of the YAML so it should be able to change/add the flag here: https://github.com/tektoncd/triggers/blob/master/config/controller.yaml#L62

Would that work?

Reply:
Yeah @savitaashture I think upstream needs to default to being able to function securely out of the box, which means being able to function if PSP is set to more restrictive default policies like what @MarcelMue and Jose experienced.

Which means default to true, no?

And then if openshift has to tweak more yaml like we do already when dealing with SCC vs. PSP, that that is what we have to do.

@vdemeester @piyush-garg FYI

Reply:
Yes :+1:

Reply:
Agreed, default to true is ok for me 👍🏼

Reply:
We will need to modify release.yaml for openshift because controller and webhook also have hardcoded UUID. So we can do for this flag also. We already do this for pipelines also.

Reply:
The *only* downside of the flag approach (compared to a configmap approach) is that a user that wants to run `upstream` triggers on OpenShift needs to edit the deployment, where, otherwise only a configmap edition is required (which is.. simpler..). But I am definitely fine with the flag 😉

Reply:
yeah in general, the number of flags are increasing...so it'd be nice to move to a configMap at some point.